zebrad/src/components/mempool.rs

```diff
@@ -382,9 +392,11 @@ fn remove_expired_transactions(
     storage.remove_same_effects(&txid_set);
 
     // also reject it
-    for id in unmined_id_set {
+    for id in unmined_id_set.clone() {
```

Comment:
Efficiency Nitpick: we don't need to clone the list here.

Since `UnminedTxId` is `Copy`, the `iter` references should automatically get dereferenced and copied.

```suggestion
    for id in unmined_id_set.iter() {
```

Reply:
done as part of https://github.com/ZcashFoundation/zebra/pull/2858/commits/16f8ea89f2d629fa2e16248cfcbb59e1c29112dc